mailpoet/mailpoet
# Changes to the first step of the Welcome Wizard [MAILPOET-4815]

## Description

This PR implements a few changes to the first step of the Welcome Wizard as described in the ticket.

## Code review notes

There is some repetition in `welcome_wizard_controller.jsx` and `woocommerce_controller.jsx` (`finishWizard()` and `updateSettings()`). I did not address this as the repetition was already there before I started this PR and it was not clear to me how to address it in a React way. Please let me know if you have any suggestions and if you think I should change this part.

## QA notes

Go to /wp-admin/admin.php?page=mailpoet-welcome-wizard#/steps/1 to access the first step of the Welcome Wizard and see the changes proposed in this PR.

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-4815]

## After-merge notes

_N/A_


[MAILPOET-4815]: https://mailpoet.atlassian.net/browse/MAILPOET-4815?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
